src/robusta/core/playbooks/playbooks_event_handler_impl.py

```diff
@@ -79,6 +79,10 @@ def run_actions(
 
```

Comment:
unrelated to your code i had a question about line 39
`if execution_event:  # might not exist for unsupported k8s types`
which ones do we not support?

Reply:
We don't support CronJobs, Secrets, Ingresses
k8s has many resource types. You can run `kubectl api-resources` to see
In addition, we don't support CRDs yet
The list of our supported types is in `generate_kubernetes_code.py`